includes/class-coblocks-site-design.php

```diff
@@ -83,29 +83,6 @@ public function __construct() {
 
 		add_action( 'wp_ajax_site_design_update_design_style', array( $this, 'update_design_style' ) );
 		add_action( 'rest_api_init', array( $this, 'design_endpoint' ) );
-
-		/**
-		 * Add the shared styles to the editor
-		 */
-		add_action(
-			'admin_init',
-			function() {
-				$suffix = SCRIPT_DEBUG ? '' : '.min';
-				$rtl    = ! is_rtl() ? '' : '-rtl';
-				// Enqueue  shared editor styles.
-				add_editor_style(
-					"dist/css/style-editor{$rtl}{$suffix}.css"
-				);
-			}
-		);
```

Comment:
We have duplicate styles! So this is superfluously loaded from here as well as from Go. Considering Site Design requires Go we can remove this functionality from CoBlocks. Also logically speaking loading this sheet from Go makes more sense anyway.